README.md

```diff
@@ -155,19 +254,25 @@ npm run build && npm run deploy
 | `S3_BUCKET_NAME`         | `''`    | S3 bucket name to deploy into                    |
 
 
-### Current issues with the development
+## Current issues with developing scratch-www
+
 We're currently in the process of transitioning into this web client from Scratch's existing structure. As we transition, there are going to be some issues along the way that relate to how this client needs to interact with the existing infrastructure to work properly in production.
 
-#### FALLBACK
+### FALLBACK
+
 On top of migrating to using this as our web client, Scratch is also transitioning into using a new API backend, Scratch REST API. As that is also currently in development and incomplete, we are set up to fall back to using existing Scratch endpoints if an API endpoint does not exist – which is where the `FALLBACK` comes in.
 
 Most of the issues we have currently revolve around the use of `FALLBACK`. This variable is used to specify what URL to fall back onto should a request fail within the context of this web client, or when using the `API_HOST`. If not specified in the process, it will not be used, and any request that is not made through the web client or the API will be unreachable.
 
 Setting `FALLBACK=https://scratch.mit.edu` allows the web client to retrieve data from the Scratch website in your development environment. However, because of security concerns, trying to send data to Scratch through your development environment won't work. This means the following things will be broken for the time being:
+
 * Login on the splash page (*In the process of being fixed*)
 * Some update attempts to production data made through a development version of the web client
 
 Additionally, if you set `FALLBACK=https://scratch.mit.edu`, be aware that clicking on links to parts of the website not yet migrated over (currently such as `Explore`, `Discuss`, `Profile`, etc.) will take you to the Scratch website itself.
 
-#### Windows
-Some users have experienced difficulties when trying to get our web client to work on Windows. One solution could be to use [Cygwin](https://www.cygwin.com/). If that doesn't work, you might want to use [Wubi](https://wiki.ubuntu.com/WubiGuide) (Windows XP, Vista, 7) or [Wubiuefi](https://github.com/hakuna-m/wubiuefi) (Windows 8 or higher). Wubi(uefi) is a Windows Installer for Ubuntu that allows you to have Ubuntu and Windows on one disk, without the need of an extra partition.
+### Windows
+
+Some users have experienced difficulties when trying to get our web client to work on Windows.
+
+One solution could be to use [Cygwin](https://www.cygwin.com/). If that doesn't work, you might want to use [Wubi](https://wiki.ubuntu.com/WubiGuide) (Windows XP, Vista, 7) or [Wubiuefi](https://github.com/hakuna-m/wubiuefi) (Windows 8 or higher). Wubi(uefi) is a Windows Installer for Ubuntu that allows you to have Ubuntu and Windows on one disk, without the need of an extra partition.
```

Comment:
WSL might help with Windows builds and could be good to mention

Reply:
Thanks! I don't know much about that. Want to suggest some text we can put here, or want to submit a PR with the change?

Reply:
I followed [this tutorial](https://docs.microsoft.com/en-us/windows/wsl/install-win10) to install WSL and then [this section from another tutorial](https://docs.microsoft.com/en-us/windows/nodejs/setup-on-wsl2#install-nvm-nodejs-and-npm) to install Node under it. The former tutorial would be a good drop-in replacement for Cygwin:

```markdown
One solution could be to use the [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/install-win10) to run Linux inside Windows.
```

Ideally, there'd be a unified setup guide that goes over installing both WSL and Node but doesn't bother with the Insider Preview stuff from the second tutorial. I may make a PR that adds such a guide.

Reply:
Integrated this. LMK if you think the result makes sense...

Reply:
Looks good :+1: